buf/registry/plugin/v1/plugin_service.proto

```diff
@@ -0,0 +1,156 @@
+// Copyright 2023-2024 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.registry.plugin.v1;
+
+import "buf/registry/owner/v1/owner.proto";
+import "buf/registry/plugin/v1/plugin.proto";
+import "buf/validate/validate.proto";
+
+option go_package = "buf.build/gen/go/bufbuild/registry/protocolbuffers/go/buf/registry/plugin/v1";
+
+// Operate on Plugins.
+service PluginService {
```

Comment:
Not sure if we're planning to bring in the [ResolveService](https://buf.build/bufbuild/buf/docs/main:buf.alpha.registry.v1alpha1#buf.alpha.registry.v1alpha1.ResolveService), but it'd be nice if we stubbed out a single RPC to resolve Generated SDK version. Something like

```
rpc ResolveGeneratedSDKVersion
```

Basically folding all the `Get{Go,Swift,Maven,NPM,...}Version` into a single RPC. You could argue that this fits nicely with the "PluginService".

Reply:
Ideally there's also an RPC to get a registry-specific package ID. Could be the same RPC.